content/ar/courses/350/blog-project/improved-design.md

```diff
@@ -1,23 +1,51 @@
 ---
-# Page title
+# Page title عنوان الصفحة 
 title: ISOM 350 - Business Application Development
 
-# Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
+عنوان :ISOM 350 تطوير تطبيقات الاعمال 
+  
+
+
+
+
+# Title for the menu link if you wish to use a shorter link title, otherwise remove thisption.
+عنوان  لقائمة الارتباط اذا كنت تتمنى استخدام عنوان رابط 
+قصير   بخلاف ذالك ازالة هذا الخيار
+
+
+
+
 linktitle: Improved Design
+عنوان الارتباط: تحسين التصميم
+
+
+# Page summary for search engines.# ملخص صفحة لمحركات البحث 
 
-# Page summary for search engines.
 summary: Second programming course for MIS majors utilizing Python to build data-driven business applications.
 
-# Date page published
+ملخص: الاستفادة من البايثون لي كورس الثاني للبرمجة  تخصص ادارة نظام المعلومات  لي  انشاء   تطبيقات الاعمال  استناد الي  بيانات
+
+
+# Date page published #تاريخ نشر الصفحة 
+
 date: 2021-03-23
+تاريخ :23-03-2021 
 
 # Academic page type (do not modify).
+#(لا تقم بالتعديل)نوع صفحة الاكاديمية
 type: book
+نوع: كتاب
 
 # Position of this page in the menu. Remove this option to sort alphabetically.
+
+ موقع هذه الصفحة في القائمة . ازالة هذا الخيار للفرز  الابجدي
+
 weight: 9
+الوزن :9
 
 draft: True
+مسودة : صح
+
```

Comment:
well done @dalalrashid the fix to the renamed file is done
however, please note everything above the --- mark should not be translated because it is data used by the system and must be in english

the only two line you can translate are title and linktitle and they should be like this:

```
title :ISOM 350 تطوير تطبيقات الاعمال 
linktitle: تحسين التصميم
```
everything else should remain in english